src/Administration/Resources/app/administration/src/module/sw-users-permissions/components/sw-user-saas-access-key-create-modal/sw-user-saas-access-key-create-modal.spec.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * @internal
+ * @sw-package after-sales
+ */
+import { mount } from '@vue/test-utils';
+
+function createDefaultProps() {
+    return {
+        isLoading: false,
+        isOpen: true,
+        accessKey: 'abdcefghi',
+        secretAccessKey: '123456789',
+        mode: 'view',
+    };
+}
+
+async function createWrapper(props) {
+    const wrapper = mount(
+        await wrapTestComponent('sw-user-saas-access-key-create-modal', {
+            sync: true,
+        }),
+        {
+            global: {
+                stubs: {
+                    'sw-modal': await wrapTestComponent('sw-modal', {
+                        sync: true,
+                    }),
+                    'sw-loader': true,
+                },
+                provide: {
+                    shortcutService: {
+                        startEventListener: () => {},
+                        stopEventListener: () => {},
+                    },
+                },
+            },
+            props: {
```

Comment:
```suggestion
            props,
```

the props look all the same, so just pass